Makefile.old

```diff
@@ -21,9 +21,16 @@ default: build
 all: build
 always:
 
-build: js-make qt-make pp-build services-build ui-build model-build conf/dictionary/words.sqlite \
+build: makedirs js-make qt-make pp-build services-build ui-build model-build conf/dictionary/words.sqlite \
 	conf/dictionary/WordsAbridged.sqlite osmapidb
 
+makedirs: 
+	mkdir -p $$HOOT_HOME/userfiles/customscript
+	mkdir -p $$HOOT_HOME/userfiles/ingest/processed/BASEMAP
+	mkdir -p $$HOOT_HOME/userfiles/ingest/upload/BASEMAP
+	mkdir -p $$HOOT_HOME/userfiles/reports
+	mkdir -p $$HOOT_HOME/userfiles/tmp/upload
+
```

Comment:
This is a great solution to the issues of creating the directories when needed and making sure that they have the proper privileges.  I like it.